src/electrum.rs

```diff
@@ -275,6 +275,88 @@ impl Rpc {
         Ok(status)
     }
 
+    fn outpoint_subscribe(&self, (txid, vout): (Txid, u32)) -> Result<Value> {
+        let funding = OutPoint { txid, vout };
+
+        let funding_blockhash = self.tracker.get_blockhash_by_txid(funding.txid);
+        let spending_blockhash = self.tracker.get_blockhash_spending_by_outpoint(funding);
+
+        let mut inputs_funding_confirmed = true;
+
+        let funding_tx = self.daemon.get_transaction(&funding.txid, funding_blockhash);
+
+        if funding_tx.is_err() {
+            return Ok(json!({}));
+        }
+
+        let funding_inputs = &funding_tx.as_ref().unwrap().input;
+        let outpoint_script = &funding_tx.as_ref().unwrap().output.get(vout as usize).unwrap().script_pubkey;
```

Comment:
I will not hide that I am terrible at Rust coding since I am litteraly doing it for the first time while working on this PR x')

I need to return an empty json if the funding transaction doesn't exist according to Electrum 1.5 specifications.  Is it handled with your suggested code ? 

Good catch for the possible out-of-bounds vout, thanks !